int32 gnss_cmd
int32 ui_cmd
int32 algorithm_cmd
int32 controller_cmd
int32 vlp16_cmd
int32 beixing_cmd
int32 ins_cmd

